### ISSUE TITLE
Build out Spotify Player admin page

### ISSUE THREAD

**Issue by IkeHunter** [ufosc/Jukebox-Frontend#87]
## Task

Build out the following page for in the admin dashboard:
- [x] Spotify Player (Admin > Player)

It's fine to only use fake data. Connecting this page to the global "store" will be a later task. Also, the connect button doesn't need to do anything yet.

![Spotify Player - Light](https://github.com/user-attachments/assets/6cbc4a50-b817-4047-a0a9-077fcd3e443f)
![Spotify Player - Dark](https://github.com/user-attachments/assets/e0dfa15a-60e4-429e-a39b-29a7e161d947)

**Comment by BaisonBeast** [comment]
Hi, I want to try this build

**Comment by BaisonBeast** [comment]
Hi as per my understanding, I have to create a page of Spotify player in the admin > pages correct?

**Comment by IkeHunter** [comment]
@BaisonBeast Yep, that's correct - and if you're using/creating any components that can be reused, you can put those in admin > components

**Comment by Ryan-slither** [comment]
I am going to start working on this

**Comment by IkeHunter** [comment]
Closing this issue since most of the features are implemented. The page will probably be redesigned in the near future though.
